__tests__/app.test.js

```diff
@@ -61,43 +61,186 @@ describe('GET /api', () => {
       });
     })
 })
-// describe('GET /api/articles/:article_id', () => {
+describe('GET /api/articles/:article_id', () => {
+  describe('Basic request checks', () => {
+    test('returns status 200 on successful request', () => {
+      return request(app).get('/api/articles/1').expect(200);
+    });
+    test('response contains an object of 1 article with correct properties', () => {
+      return request(app)
+      .get('/api/articles/1')
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 1,
+          title: 'Living in the shadow of a great man',
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'I find this existence challenging',
+          created_at: '2020-07-09T20:11:00.000Z',
+          votes: 100,
+          article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        })
+      });
+    });
+  });
+  describe('errors', () => {
+        test('should respond with 404 when article is not found', () => {
+          return request(app)
+            .get('/api/articles/99999999')
+            .expect(404)
+            .then(({ body }) => {
+              expect(body).toEqual({ msg: 'Article not found' });
+            });
+        });
+        test('should respond with 400 when invalid endpoint is used', () => {
+          return request(app)
+          .get('/api/articles/not_an_id')
+          .expect(400)
+          .then(({ body }) => {
+          expect(body.msg).toEqual("invalid input syntax for type integer: \"not_an_id\"" || 'Bad request');
```

Comment:
The message for this bad path must only be "Bad path" The user won't find the "invalid input... " message useful at all.